server/src/main/java/org/apache/iotdb/db/metadata/mnode/StorageGroupMNode.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.metadata.mnode;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.apache.iotdb.tsfile.write.schema.MeasurementSchema;
+
+/**
+ * This class is the implementation of Metadata Node. One MNode instance represents one node in the
+ * Metadata Tree
+ */
+public class StorageGroupMNode extends InternalMNode {
+
+  private static final long serialVersionUID = 7999036474525817732L;
```

Comment:
Because MNode implements `Serializable` interface and without a serialVersionUID field, any change to a class will make previously serialized versions unreadable.